### PR TITLE
New Docs Website / Support page

### PR DESCRIPTION
**Pull request by MelSumner**
### :pushpin: Summary

If merged, this PR adds content to the `Support` page.

### :link: External links
Jira ticket: [HDS-1072](https://hashicorp.atlassian.net/browse/HDS-1072)

***

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

### CODE REVIEW

**Comment by MelSumner** [comment]
Left some questions in the corresponding JIRA ticket, need some additional info before I can move this forward.

File: website/app/templates/support.hbs [hunk 1]
```diff
@@ -1,8 +1,20 @@
 {{page-title 'Support'}}
 
 <Doc::Page::Stage>
+  <Doc::Page::Cover @title="Support" />
   <Doc::Page::Content>
-    <h1>Support</h1>
-    <p>TBD what this page will contain</p>
+    <p class="doc-text-body">We are happy to provide support to HashiCorp product teams that use our design system.Here are some of the ways to get help when you get stuck.</p>
```
Comment (heatherlarsen) [inline website/app/templates/support.hbs:6]:
```suggestion
    <p class="doc-text-body">We're happy to support HashiCorp product teams that use our design system. Here are some of the ways to get help when you get stuck.</p>
```

**Review by heatherlarsen — APPROVED** [review]
Content is great! 🙂 🎉

File: website/app/templates/support.hbs (hunk 1 above)
Comment (KristinLBradley) [inline website/app/templates/support.hbs]: Missing a space in between the sentences, also I would say "if" they get stuck vs. "when" to sound more positive:

"We are happy to provide support to HashiCorp product teams that use our design system.Here are some of the ways to get help when you get stuck."
=>
"We are happy to provide support to HashiCorp product teams that use our design system. Here are some of the ways to get help if you get stuck."

**Review by KristinLBradley — APPROVED** [review]

**Comment by MelSumner** [comment]
@heatherlarsen good catch, fixed it!

**Pull request merged**